src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -262,17 +262,17 @@ export class MenuEntryActionViewItem extends ActionViewItem {
 					iconClass = MenuEntryActionViewItem.ICON_PATH_TO_CSS_RULES.get(iconPathMapKey)!;
 				} else {
 					iconClass = ids.nextId();
-					createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
-					createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.dark)}`);
+					createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
```

Comment:
If you're making changes to core VS code you should be adding a SQL CARBON EDIT to the affected lines

But I'm not sure about making the change here - do you have a clear idea of what side effects this might cause? I dug through the code a little bit and it doesn't seem like it's used outside the debug viewlet but I might have missed something. We should be careful making core changes like this in case we break other things in the process.

I tested just making the codicon changes to the LabeledMenuItemActionItem class below and that seems to fix the issue and is much more specific to this case (the task bar instead of a general Menu item). I think it'd be better to revert the changes to this base class and instead just make them in the LabeledMenuItemActionItem class.

Reply:
Alright I'll revert the ones for the core classes, but searching around the code, I did not see any rules for .icon so far, and it didn't break anything when I tested it, also I'll add the SQL CARBON EDIT as well.